pkg/piecestore/rpc/client/readerwriter.go

```diff
@@ -94,22 +97,10 @@ func NewStreamReader(signer *Client, stream pb.PieceStoreRoutes_RetrieveClient,
 	}
 
 	go func() {
```

Comment:
add comment here what this go routine does or put it in a separate func on stream reader, so this line would be: `go sr.sendSignedAllocations(...)`